budget-api/src/utils/source-typeorm.utils.ts

```diff
@@ -0,0 +1,18 @@
+import { Source } from "../models/source.model";
+import { Equal, FindManyOptions, FindManyOptions as FindOneOptions, In } from "typeorm";
+
+export class SourceTypeormUtils {
+  public static findOneByName(name: string): FindOneOptions<Source> {
+    const options: FindOneOptions<Source> = {
+      where: { name: Equal(name) }
+    }
+    return options;
+  }
+
+  public static findManyByNames(names: string[]): FindManyOptions<Source> {
+    const options: FindManyOptions<Source> = {
+      where: {name: In(names)}
```

Comment:
Let's make spaces in all objects within curly braces equal: `{ name: In(names) }` :)